crates/bevy_window/Cargo.toml

```diff
@@ -21,3 +21,6 @@ bevy_utils = { path = "../bevy_utils", version = "0.1" }
 
 # other
 uuid = { version = "0.8", features = ["v4", "serde"] }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+uuid = { version = "0.8", features = ["wasm-bindgen"] }
```

Comment:
does this merge the wasm features with the `["v4", "serde"]` features?

Reply:
Yes. Features are additive.